WooCommerce/Classes/ViewRelated/Orders/Order Creation/NewOrder.swift

```diff
@@ -45,11 +45,17 @@ struct NewOrder: View {
     @ObservedObject var viewModel: NewOrderViewModel
 
     var body: some View {
-        ScrollView {
-            EmptyView()
-        }
-        .background(Color(.listBackground))
+        GeometryReader { geometry in
```

Comment:
Apparently there is a new `safeAreaInset()` modifier to help with this use case but it's only available on iOS15 😢

Reply:
TIL! Too bad it's only iOS15.